components/FeedbackCard/Feedback-card.tsx

```diff
@@ -0,0 +1,221 @@
+"use client";
+import Carousel from "@/components/Carousel/Carousel";
+import {
+  Box,
+  Card,
+  Flex,
+  GridItem,
+  Heading,
+  HStack,
+  Image,
+} from "@chakra-ui/react";
+import { Avatar } from "@/components/ui/avatar";
+import React from "react";
+export default function FeedbackCard() {
+  const cards = [
+    {
+      id: "1",
+      title: "اسم المستخدم",
+      content:
+        "تقدم الأكاديمية العربية للبرمجة تجربة تعلم متميزة من خلال مجموعة من الدروس والمناهج الاحترافية بجودة عالية وأسلوب تدريسي ممتع يتناسب",
+      month: "نيسان",
+      date: " 11, 2023",
+
+      image: "card1.png",
+      curve: "curve.png",
+    },
+    {
+      id: "2",
+      title: "اسم المستخدم",
+      content:
+        "تقدم الأكاديمية العربية للبرمجة تجربة تعلم متميزة من خلال مجموعة من الدروس والمناهج الاحترافية بجودة عالية وأسلوب تدريسي ممتع يتناسب",
+      date: " 11, 2023",
+      month: "نيسان",
+      image: "card2.png",
+      curve: "curve.png",
+    },
+    {
+      id: "3",
+      title: "اسم المستخدم",
+      content:
+        "تقدم الأكاديمية العربية للبرمجة تجربة تعلم متميزة من خلال مجموعة من الدروس والمناهج الاحترافية بجودة عالية وأسلوب تدريسي ممتع يتناسب",
+      date: " 11, 2023",
+      month: "نيسان",
+      image: "card3.png",
+      curve: "curve.png",
+    },
+  ];
+  return (
+    <div>
+      <Box
+        paddingX={{
+          base: "16",
+          md: "28",
+        }}
+        my={{
+          lg: 20,
+          md: 30,
+          base: 15,
+        }}
+      >
+        <Carousel
+          slidesToShow={{
+            base: 1,
+            md: 2,
+            lg: 3,
+          }}
+          arrowGaps={{
+            base: "-40px",
+            md: "-80px",
+            lg: "-90px",
+          }}
+        >
+          {cards.map((item) => (
```

Comment:
why all cards are rendered in the feedback card component, the component suppose to just render one card